.github/workflows/ecosystem.yml

```diff
@@ -0,0 +1,43 @@
+name: Test Ecosystem Plugins
+
+on:
+    push:
+        branches:
+            - "ecosystem/*"
```

Comment:
[Feature] I hadn't thought of having a branch wildcard in the RFC. But I think it'd be nice if we want to have branches/PRs that explicitly test ecosystem logic. Especially for updating this file.

https://github.com/JoshuaKGoldberg/eslint/pull/19

Reply:
I'm not sure there's much value in doing this on `push`. Maybe on `pull_request` would make more sense?

Reply:
I was thinking that if someone wants to work on the ecosystem flow itself, it would be inconvenient to have to always manually dispatch the workflow. This `branches` setting means the flow will run on every push to a branch with a name matching `ecosystem/*`.